tests/integration/internal/conftest.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from spccore.internal.cache import *
+
+
+TEST_CACHE_ROOT_DIR = "cache_integration_test"
+
+
+@pytest.fixture
+def cache() -> Cache:
+    os.makedirs(TEST_CACHE_ROOT_DIR)
+    cache = Cache()
```

Comment:
didn't specify directory during creation of cache
`Cache(cache_root_dir=TEST_CACHE_ROOT_DIR)`